File: C1.UWP.FlexChart/VB/FlexChartMvvmDemo/Model/OrderModel.vb
Imports System.Threading.Tasks
Imports System.Text
Imports System.Linq
Imports System.Collections.Generic
Imports System

Namespace Model
    Public Class OrderModel
        Public Property ID() As Integer

        Public Property [Date]() As DateTime

        Public Property Category() As String

        Public Property Country() As String

        Public Property Year() As String

        Public Property Amount() As Double
    End Class
End Namespace

File: C1.UWP.FlexChart/CS/FlexChartMvvmDemo/Model/OrderModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FlexChartMvvmDemo.Model
{
    public class OrderModel
    {
        public int ID
        {
            get; set;
        }

        public DateTime Date
        {
            get; set;
        }

        public string Category
        {
            get;
            set;
        }

        public string Country
        {
            get;
            set;
        }

        public string Year
        {
            get;
            set;
        }

        public double Amount
        {
            get;
            set;
        }
    }
}
